### ISSUE TITLE
`pamac-flatpak` vs `pamac-nosnap`

### ISSUE THREAD

**Issue by saltedcoffii** [tommytran732/pamac-flatpak#4]
hey! I'm the maintainer of `pamac-nosnap`, and I was wondering why you forked the package, and if there's anything I can merge upstream to make the package better.

**Comment by tommytran732** [comment]
> hey! I'm the maintainer of `pamac-nosnap`, and I was wondering why you forked the package, and if there's anything I can merge upstream to make the package better.

No, there's no substantial differences. I made this package last year because having snap with `pamac-all` was very annoying (before `pamac-nosnap` existed), but then I went inactive. I just came back recently, and I noticed `pamac-nosnap` exists so I decided to just use it as a base.

There are a few small changes that I am not sure whether you are willing to merge or not:
- flathub-beta is added by default 
- it doesn't persistently add flathub back every upgrade if the user remove it

**Comment by saltedcoffii** [comment]
Okay cool that makes sense!

- I'm not sure if I want to add `flathub-beta` by default due to stability concerns, I think a user that wants it can add it manually. I think that's reasonable, do you?
- Yes, I will make `pamac-nosnap` so that it won't forcibly add `flathub` (stable) back on every update when/if the user removes

**Comment by saltedcoffii** [comment]
After I made the aforementioned change, would you like me to add you as a co-maintainer of `libpamac-nosnap` and `pamac-nosnap`? After I add you, we can issue a merge request for `libpamac-nosnap`:`libpamac-flatpak` and `pamac-nosnap`:`pamac-flatpak`? Let me know! And if yes, let me know what your AUR username is.

**Comment by tommytran732** [comment]
> After I made the aforementioned change, would you like me to add you as a co-maintainer of `libpamac-nosnap` and `pamac-nosnap`? After I add you, we can issue a merge request for `libpamac-nosnap`:`libpamac-flatpak` and `pamac-nosnap`:`pamac-flatpak`? Let me know! And if yes, let me know what your AUR username is.

Sure, that works! My AUR username is the same as here: tommytran732

**Comment by tommytran732** [comment]
Actually, I want to ask what is your plan for pamac-flatpak-gnome? It's pretty much the same thing but with fake-gnome-software enabled

**Comment by saltedcoffii** [comment]
the pamac-nosnap package has fake gnome software support enabled, and for those not using gnome, it won't affect them.

**Comment by tommytran732** [comment]
> the pamac-nosnap package has fake gnome software support enabled, and for those not using gnome, it won't affect them. do you think that's reasonable?

Not sure if that's the best idea, what if a GNOME user for some reason wanna keep both gnome-software and pamac-nosnap? It's not like pamac has things like fwupd integration like gnome-software does. Idk, what do you think?

**Comment by saltedcoffii** [comment]
ah. hmm. fwupd is important. maybe having `pamac-nosnap-gnome` that just packages the gnome software integration and depends on `pamac-nosnap`?

**Comment by tommytran732** [comment]
> ah. hmm. fwupd is important. maybe having `pamac-nosnap-gnome` that just packages the gnome software integration and depends on `pamac-nosnap`?

Sure I guess. Let's do that then.

**Comment by tommytran732** [comment]
@saltedcoffii Can you update me on this?

If talking through GitHub issue is too inconvenient, you can contact me using Signal/Matrix/Email/Twitter. My accounts are listed here: https://tommytran.io/contact/
